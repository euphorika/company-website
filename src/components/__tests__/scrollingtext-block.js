import React from "react"
import renderer from "react-test-renderer"

import ScrollingTextBlock from "../scrollingtext-block"

describe("ScrollingTextBlock", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ScrollingTextBlock title="Awesome Headline">Awesome Content</ScrollingTextBlock>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
