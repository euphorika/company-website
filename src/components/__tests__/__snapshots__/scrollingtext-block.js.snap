// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollingTextBlock renders correctly 1`] = `
<div
  className="scrollingTextBlockContainer"
>
  <h3
    dangerouslySetInnerHTML={
      Object {
        "__html": "Awesome Headline",
      }
    }
  />
  Awesome Content
</div>
`;
